v2/thirdparty/custom-ui/multitenant-login.mdx

```diff
@@ -0,0 +1,93 @@
+---
+id: multitenant-login
+title: "Multitenant login"
+hide_title: true
+---
+
+import AppInfoForm from "/src/components/appInfoForm"
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
+import GoFrameworkSubTabs from "/src/components/tabs/GoFrameworkSubTabs";
+import TabItem from '@theme/TabItem';
+import MultitenancyIntro from '../reusableMD/multitenancy/intro.mdx'
+import CreateConfigureTenants from '../reusableMD/multitenancy/create-configure-tenants.mdx'
+import BackendCORS from '../reusableMD/multitenancy/backend-cors.mdx'
+
+:::important
+TODO BANNER This is a paid feature
+
+If you wish to use this feature for free, follow this implementation guide TODO link
+:::
+
+# Multitenant login
+
+<MultitenancyIntro />
+
+<CreateConfigureTenants />
+
+## Step 2: Frontent setup
+
+To fetch the login methods dynamically from the backend, use the following API:
+
+```tsx
+
+import SuperTokens from 'supertokens-web-js';
+
+const loginMethods = await SuperTokens.getLoginMethods({
+  tenantId: '...'
+})
+
+if (loginMethods.thirdParty.enabled) {
+  const providers = loginMethods.thirdParty.providers;
+  // render the login buttons
+} else {
+  // thirdparty login is disabled for the tenant
+}
+```
+
```

Comment:
mention what happens next -> when you call getAuthorizationUrl function, pass in the tenantId to it. SignInUp function remans the same. This applies for web. For mobile, say that tenantId needs to be passed to sign in up API call as well.